contract_show_invoice/__manifest__.py

```diff
@@ -9,6 +10,7 @@
     'version': '10.0.1.0.0',
     'author': 'Domatix,'
               'Tecnativa,'
+              'Thinkwell Designs,'
```

Comment:
Authorship and copyright is a bit of an overreach for the work that was performed here. Please limit the attribution to the ReadMe in this instance